test/parallel/test-https-autoselectfamily-slow-timeout.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const { request } = require('https');
+
+request(
+  'https://nodejs.org/en',
```

Comment:
If this is reaching out to the internet this should go in `internet` and not `parallel`. It should also be using a constant from [common/internet](https://github.com/nodejs/node/tree/main/test/common#internet-module) to allow people to run the test suites in places that restrict access to some addresses.

Reply:
Thanks, I didn't know about this. This is fixed now.